spec/rails_helper.rb

```diff
@@ -95,9 +95,12 @@
     WebMock.disable_net_connect!(allow_localhost: true)
   end
 
-  # This is to prevent test from having to mock this callback due
+  # This is to prevent most tests from having to mock this callback due
   # to an after_save call back in Allocation.
-  config.before(:each, :allocation) do
+  #config.before(:each, type: lambda { |_v, m| puts "m=#{m.inspect}"; m[:push_pom_to_delius] != true } ) do
+  #  allow(PushPomToDeliusJob).to receive(:perform_later)
+  #end
+  config.before(:each, type: lambda { |_v, m| m[:push_pom_to_delius] != true } ) do
```

Comment:
Might be helpful to add a comment explaining how to use this new 'opposite' tag. You've explained that it's mocked by default, but how do I disable the mock if I want to be able to test pushing a POM to nDelius?

Should reduce the need to interpret the meaning of the lambda.